Client/src/components/contanier/staff/Student/EditStudent/index.js

```diff
@@ -0,0 +1,93 @@
+import React, { Component } from 'react'
+import LabeledInput from './../../../../common/LabeledInput'
+import Button from './../../../../common/Button';
+import { studentSchema } from './../../../../../helpers/validation-schema'
+import { leftSectionInfo, rightSectionInfo } from './staticdata';
+
+import './index.css';
+
+export default class EditStudent extends Component {
+  state = {
+    firstname: '',
+    address: '',
+    level: 0,
+    phonenumber: '',
+    lastname: '',
+    mobilenumber: '',
+    password: '',
+    Error: {}
+  }
+  handleChange = ({ target: { value, name } }) => {
+    if (name === 'level') {
+      if (!isNaN(Number(value))) {
+        this.setState({ [name]: value });
+      }
+    } else this.setState({ [name]: value });
+  }
+
+  handleClick = (e) => {
+    e.preventDefault();
+    this.setState({ Error: {} })
+    const { Error, ...student } = this.state;
+    studentSchema
+      .validate(student, {
+        abortEarly: false
+      })
+      .then((value) => {
+        // write fetch here 
+        console.log(value)
+      })
+      .catch(({ inner, fetchError }) => {
+        if (fetchError) {
+          // handle fetch Error
+        }
+        if (inner) {
+          const errors = inner.reduce((acc, item) => ({ ...acc, [item.path]: (item.message) }), {});
+          this.setState({ Error: { ...errors } })
+        }
+      })
+  }
+
+  render() {
+    return (
+      <div className='edit-student'>
+        <h1 className='edit-student-title'>
+          Edit Student
+        </h1>
+        <form className='edit-student-contanier'>
+          <div className='edit-student-contanier-left'>
+            {
+
+              leftSectionInfo.map(info => {
```

Comment:
@Fatmasiam One thing that might help the code readability here is if you changed your staticdata.js file to be arrays of objects, like this:
```
const leftSectionInfo = [
  {id: 'firstname', displayName: 'First name'},
  {id: 'lastname', displayName: 'Last name'},
  ...
```

Then you could destructure in your map here and have more meaningful names:
```
leftSectionInfo.map(({id, displayName}) => {
  return <LabeledInput
    labelText={displayName}
    id={id}
    name={id}
...
```
```